.eslintignore

```diff
@@ -1,4 +1,5 @@
 coverage/
+examples/
```

Comment:
We can just put another `.eslintrc.json` file in the `examples` directory that changes to ES5 settings?